pkgs/development/python-modules/oauthlib/default.nix

```diff
@@ -38,6 +45,15 @@ buildPythonPackage rec {
     "oauthlib"
   ];
 
+  passthru.tests = {
+    inherit
+      django-allauth
+      django-oauth-toolkit
+      google-auth-oauthlib
+      jupyterhub
```

Comment:
Currently marked broken and would be inconvenient to have ofborg abort due to that. Feel free to remove and self-merge.